src/Robo/Inspector/Inspector.php

```diff
@@ -761,4 +761,19 @@ protected function warnIfXdebugLoaded() {
     }
   }
 
+  /**
+   * Determines if the active config is identical to sync directory.
+   *
+   * @return bool
+   *   TRUE if config is identical.
+   */
+  public function isActiveConfigIdentical() {
+    $result = $this->executor->drush("cex --no")
+      ->run();
+    $message = trim($result->getMessage());
+    if (strpos($message, 'The active configuration is identical to the configuration in the export directory')) {
```

Comment:
I think I'd rather rely on the exit code. Will `$result->getExitCode()` suffice?

Reply:
@grasmash it seems like `$result->getExitCode()` won't work here. Even if the active config matches the export dir config, that results in an exit code of `1` since the config isn't exported.